source/distributed/tria.cc

```diff
@@ -2530,10 +2700,12 @@ namespace parallel
         {
           std::string   fname = std::string(filename) + ".info";
           std::ofstream f(fname.c_str());
-          f << "version nproc n_attached_objs n_coarse_cells" << std::endl
-            << 3 << " "
+          f << "version nproc n_attached_fixed_size_objs n_attached_variable_size_objs n_coarse_cells"
+            << std::endl
+            << 4 << " "
```

Comment:
We do not have to increase the version number here, since it has not changed since the last release (in which it was `2`). What do you think?

Reply:
It doesn't really matter. Version 3 only existed for a short time but there is no disadvantage to making it 4 now. I would say leave it as 4.

Reply:
Yes, that's reasonable.